src/main/java/com/exercise/system/exercisesystem/services/CrudService.java

```diff
@@ -0,0 +1,13 @@
+package com.exercise.system.exercisesystem.services;
+
+import java.util.Set;
+
+public interface CrudService<T, ID> {
```

Comment:
let's consider to drop intermediate layer between Data Access layer and Service layer